native/src/components/NativeHtml.tsx

```diff
@@ -86,6 +86,16 @@ const NativeHtml = React.memo(({ content, cacheDictionary, language }: PropsType
         direction: config.hasRTLScript(language) ? 'rtl' : 'ltr'
       }}
       systemFonts={fonts}
+      tagsStyles={{
+        figure: {
+          margin: 0,
+          marginBottom: 15,
+          maxWidth: '100%',
```

Comment:
couldn't override inline style of the width that comes from the `content` thats why i set the `maxWidth: 100%`